pkg/indexer/storer/groupMessage.go

```diff
@@ -13,6 +13,11 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	// We may not want to hardcode this to 0 and have an originator ID for each smart contract?
+	GROUP_MESSAGE_ORIGINATOR_ID = 0
```

Comment:
I thought it was supposted to be `>=1 ID < 100`...

Reply:
To be fair, I just moved the group message origId to its own const, but it was always hardcoded 0. Let me extract an issue out of this.